tests/integration/conftest.py

```diff
@@ -53,6 +52,7 @@
 )  # noqa
 
 
+@mock.patch('inspirehep.modules.records.receivers.batch_reindex.apply_async')
```

Comment:
maybe it's better to localize this to the migration of records, otherwise you're patching this for all tests using the `app` fixture.